samples/nrf9160/gps/README.rst

```diff
@@ -0,0 +1,126 @@
+.. _gps_with_supl_support_sample:
+
+nRF9160: GPS sockets and SUPL client library
+############################################
+
+The GPS socket sample reads the `GPS`_ data from the `GNSS`_ socket and parses the received frames to interpret the frame contents.
```

Comment:
Can we have a more general description as first sentence?
For example:

    The GPS socket sample demonstrates how to retrieve GPS data, either by using the data provided by the GPS module of the nRF9160 DK or by using A-GPS data provided by the external :ref:`supl_client` library.

    The sample reads the `GPS`_ data ...

Not sure if the description of the "simple" mode vs. A-GPS mode is correct though?

Reply:
Also, maybe we should state that the sample first establishes a GNSS socket (and link to BSDlib for information about sockets). "It then reads the GPS data from the socket and ..."

Reply:
Ok, I have changed the sentence using yours as a template. But it's not GPS or AGPS, AGPS is used to augment the GPS. Look through my update when I have pushed it.